tests/unit/test_client.py

```diff
@@ -94,7 +94,7 @@ def assert_headers(headers):
         assert headers[constants.HEADER_SESSION] == ""
```

Comment:
As the count of headers increased, please also assert the role is empty string.

```suggestion
        assert headers[constants.HEADER_SESSION] == ""
        assert headers[constants.HEADER_ROLE] == ""
```

Reply:
Done. Thanks